tutorials/applications/Microwave-engineering of programmable XXZ Hamiltonians in arrays of Rydberg atoms .ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Maybe add PBC and OBC as a title on each plot?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>